README.md

```diff
@@ -1,51 +1,81 @@
 | SFCC_CLIENT_ID        | Client id related to the account from where need to send logs. [Learn more](https://documentation.b2c.commercecloud.salesforce.com/DOC3/index.jsp?topic=%2Fcom.demandware.dochelp%2Fcontent%2Fb2c_commerce%2Ftopics%2Faccount_manager%2Fb2c_account_manager_add_api_client_id.html)                                                           |      Yes |
 | SFCC_CLIENT_SECRET    | Client secret related to the account from where need to send logs. [Learn more](https://documentation.b2c.commercecloud.salesforce.com/DOC3/index.jsp?topic=%2Fcom.demandware.dochelp%2Fcontent%2Fb2c_commerce%2Ftopics%2Faccount_manager%2Fb2c_account_manager_add_api_client_id.html)                                                       |      Yes |
+| SFCC_LOG_SOURCE       | Flag to represent which log types you would like to collect. `operational` - for operational logs, `security` for security logs, `all` for both of them.                                                                                                                                                                                      |      Yes |
+| AUTO_PARSING          | Default: `false`. To enable auto parsing (by our prebuilt grok patterns ) set to `true`.                                                                                                                                                                                                                                                      |       No |
 
-If you prefer to store these environment variables in a file like [this example](./variables.env), you can run docker like so:
+If you prefer to store these environment variables in a file like [this](./variables.env), you can run Docker as follows:
 
 ```sh
 docker run -d --env-file=variables.env logzio/webdav-fetcher:latest
 ```
 
+### 3. Grok patterns
+
+Based on [fluentD-grok-parser](fluent-plugin-grok-parser) you have option to manage grok patterns base on log types.<br/>
+Log types is: `analytics`, `api`, `codeprofiler`, `console`, `customdebug`, `customerror`, `customfatal`, `custominfo`, `customwarn`, `debug`, `deprecation`, `error`, `fatal`, `info`, `jobs`, `migration`, `performance`, `quota`, `sql`, `staging`, `sysevent`, `syslog`, `warn`,`service`, `syslog`
+
+We are prepared grok-patterns for you. You can use if define env. variable as `AUTO_PARSING` for `true` value(by default is `false`). But if you want to define your own grok patterns, you can download sample of grok patterns(file `example.grokPatternList.json`) on you folder where you will run docker image, there you can fill your grok patterns based on log types. For example:
```

Comment:
Please change to:
You can use our prebuilt grok patterns by setting the `AUTO_PARSING` environment variable to `true`. If you would like to define your own grok patterns you can do this by downloading the sample grok pattern file to the directory where you are running the docker image and edit it accordingly. For example